src/app/work-item/work-item.service.ts

```diff
@@ -17,7 +17,9 @@ export class WorkItemService {
   constructor(private http: Http,
               private logger: Logger,
               private auth: AuthenticationService) {    
-    this.headers.append('Authorization', 'Bearer ' + this.auth.getToken());
+    if(this.auth.getToken() != null) {
```

Comment:
@aslakknutsen I merged it, if you want to make that change please send a new PR once this branch is in master.